crates/tools/src/prepare_scripts.rs

```diff
@@ -0,0 +1,389 @@
+use crate::deploy_scripts::Programs;
+use anyhow::Result;
+use ckb_fixed_hash::H256;
+use ckb_jsonrpc_types::{JsonBytes, Script, ScriptHashType};
+use clap::arg_enum;
+use serde::{Deserialize, Serialize};
+use std::{
+    collections::HashMap,
+    env,
+    ffi::OsStr,
+    fs,
+    path::{Path, PathBuf},
+    process::Command,
+};
+use url::Url;
+
+const GODWOKEN_SCRIPTS: &str = "godwoken-scripts";
+const GODWOKEN_POLYJUICE: &str = "godwoken-polyjuice";
+const CLERKB: &str = "clerkb";
+
+static DEFAULT_BUILD_CONFIG: &str = r#" {
```

Comment:
Rust already has the Default trait for constructing default values. You should manually implement the trait to set the default value, do not use JSON string as the default value of struct.